### PR TITLE
handle missing content-type

### PR DESCRIPTION
**Pull request by mmmmmrob**

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -30,6 +30,9 @@ def fixture_file(filename)
 # Used to test response status codes
 FakeWeb.register_uri(:get, "http://example.com/404", :response => fixture_file("404.response"))
 
+# Used to test headers
+FakeWeb.register_uri(:get, "http://exmaple.com/no-content-type", :response => fixture_file("no-content-type.response"))
```
Comment (jaimeiniesta) [inline spec/spec_helper.rb]: exmaple.com or example.com? :smiley:

**Comment by jaimeiniesta** [comment]
:+1, just check the typo

**Comment by mmmmmrob** [comment]
typo fixed and added another needed nil fix

**Comment by jaimeiniesta** [comment]
:+1:, released 4.3.1

**Pull request merged**